tests/components/reolink/conftest.py

```diff
@@ -62,6 +62,100 @@ def mock_setup_entry() -> Generator[AsyncMock]:
         yield mock_setup_entry
 
 
+def _init_host_mock(host_mock: MagicMock) -> None:
```

Comment:
in the final PR, after removing everything that depends on autospec, I'll remove `reolink_connect_class()` and just move what's on this method to `reolink_host()`.

Reply:
seems good